src/lib.rs

```diff
@@ -12,9 +12,7 @@
 //! [`kurbo`]: https://crates.io/crates/kurbo
 
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
-// doc_markdown was updated in https://github.com/rust-lang/rust-clippy/pull/11735
-// to be more accurate, but our MSRV is earlier than that
-#![cfg_attr(not(peniko_msrv), warn(clippy::doc_markdown))]
+#![warn(clippy::doc_markdown)]
```

Comment:
Should this use the `lints` table?

Hmm, I guess that's an MSRV hazard. Maybe it's fine

Reply:
Yeah `[lints]` would bump MSRV. Not worth it right now, but we should definitely adopt `[lints]` later when MSRV allows.